lib/Cake/Model/Model.php

```diff
@@ -2895,6 +2895,10 @@ public function exists($id = null) {
 			return false;
 		}
 
+		if ($this->useTable === false) {
+			return true;
```

Comment:
I would think exists() should always return false, when useTable == false. We have no way of knowing whether or not the record exists, and often there is _no_ record so it can't exist.